src/main/java/frc/robot/lib/states/Arm.java

```diff
@@ -4,14 +4,51 @@
 
 package frc.robot.lib.states;
 
+import com.ctre.phoenix.motorcontrol.NeutralMode;
 import com.ctre.phoenix.motorcontrol.TalonFXControlMode;
+import com.ctre.phoenix.motorcontrol.TalonFXFeedbackDevice;
 
 import frc.robot.RobotMap;
 
 /** Add your docs here. */
 public class Arm {
 
-    public Arm() {}
+    public static double counter = 0.0;
+    public static double counter2 = 0.0;
+    public static double counter3 = 0.0;
+    public static double counter4 = 0.0;
+
+    public Arm() {
+        RobotMap.armFirstStage.setNeutralMode(NeutralMode.Brake);
+        RobotMap.armSecondStage.setNeutralMode(NeutralMode.Brake);
+        RobotMap.wrist.setNeutralMode(NeutralMode.Brake);
+
+        RobotMap.armFirstStage.configSelectedFeedbackSensor(TalonFXFeedbackDevice.IntegratedSensor, 0, 10);
+        RobotMap.armSecondStage.configSelectedFeedbackSensor(TalonFXFeedbackDevice.IntegratedSensor, 0, 10);
+        RobotMap.wrist.configSelectedFeedbackSensor(TalonFXFeedbackDevice.IntegratedSensor, 0, 10);
+
+        RobotMap.armFirstStage.configNominalOutputForward(0, 10);
```

Comment:
All of these values we are setting need to be constants defined in Constants.java